CHANGELOG.md

```diff
@@ -4,7 +4,203 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
-## [1.17.4](https://github.com/go-gitea/gitea/releases/tag/1.17.4) - 2022-12-21
+## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/v1.18.0) - 2022-12-29
+
+* SECURITY
+  * Remove ReverseProxy authentication from the API (#22219) (#22251)
+  * Support Go Vulnerability Management (#21139)
```

Comment:
this one is really misleading as this is no way related to Gitea itself but just PR review check. At least myself reading this at first I thought it's feature to allow manage Go project vulnerabilities in Gitea

Reply:
I think it's fine to leave here, although technically it wasn't a security fix so much as a security pre-emptive check.

Reply:
If it would be moved to build section below I would not mind but as it is now it's very misleading 🤷🏻‍♂️

Reply:
This is now how we have it on the other branches, as well as the blog. 
I am still ambivalent on this, but does anyone else have an opinion here?

If not, I think we can merge this as-is.

Reply:
I think it's fine as is - we just have to label pulls more precisely